2008-04-17  Arnaud Charlet  <charlet@adacore.com>

	* src/cairo-ps-surface.c: Define _POSIX_PTHREAD_SEMANTICS, needed
	under Solaris.
	Submitted under https://bugs.freedesktop.org/show_bug.cgi?id=15566

--- src/cairo-ps-surface.c.old	2008-04-17 15:14:36.996930195 +0200
+++ src/cairo-ps-surface.c	2008-04-17 15:15:36.527525972 +0200
@@ -41,6 +41,7 @@
  */
 
 #define _BSD_SOURCE /* for ctime_r(), snprintf(), strdup() */
+#define _POSIX_PTHREAD_SEMANTICS
 #include "cairoint.h"
 #include "cairo-ps.h"
 #include "cairo-ps-surface-private.h"
